src/VirtualClient/VirtualClient.Dependencies/MySqlServerConfiguration.cs

```diff
@@ -223,7 +223,7 @@ await this.ExecuteCommandAsync(manager, $"{WindowsMySQLPackagePath}mysql.exe --e
                     await this.ExecuteCommandAsync(manager, $"{WindowsMySQLPackagePath}mysql.exe --execute=\"CREATE USER '{this.DatabaseName}'@'{clientIp}'\" --user=root", null, telemetryContext, cancellationToken)
                         .ConfigureAwait(false);
 
-                    await this.ExecuteCommandAsync(manager, $"{WindowsMySQLPackagePath}mysql.exe --execute=\"GRANT ALL ON {this.DatabaseName}.* TO '{this.DatabaseName}'@'{clientIp}'\" --user=root", null, telemetryContext, cancellationToken)
+                    await this.ExecuteCommandAsync(manager, $"{WindowsMySQLPackagePath}mysql.exe --execute=\"GRANT ALL ON *.* TO '{this.DatabaseName}'@'{clientIp}'\" --user=root", null, telemetryContext, cancellationToken)
```

Comment:
Does this solve the database larger than 13 problem on ARM?

Reply:
this allows the client to create/recreate tables on the disks for the balanced scenario after the original sbtest db is reated